docs/Makefile

```diff
@@ -0,0 +1,17 @@
+.PHONY: docs
+docs: setup format
+
+.PHONY: setup
+setup:
+	@rm -rf mdformatter
+	@git clone https://github.com/caraml-dev/mdformatter.git
+	@pip install -r mdformatter/requirements.txt
+
+# The target below uses a non-existent doc overrides folder name to generate the final docs,
+# as there are no overrides.
+.PHONY: format
+format:
+	@echo "Formatting maintainer docs ..."
+	@cd mdformatter && python -m mdformatter ../maintainer/templates ../maintainer/overrides ../maintainer/generated ../maintainer/values.json GITBOOK
```

Comment:
`../maintainer/overrides` doesn't have to be physical there is not required as per this repo right?

Reply:
Correct, it doesn't need to be there physically. If the specified override directory is not found, the merger tool will ignore it.